ginipaybusiness/src/doc/source/index.rst

```diff
@@ -1,18 +1,21 @@
-Gini Pay Business SDK for Android
+Gini Health SDK for Android
 =================================
 
-Gini Pay provides an information extraction system for analyzing business invoices and
-transfers them to the Android banking app, where the payment process will be completed.
-The Gini Pay Business SDK for Android provides functionality to review the document and
-its extractions and pay using a payment provider.
+The Gini Health SDK for Android provides all the UI and functionality needed to use the Gini Pay API in your app to
+extract payment and health information from invoices. The payment information can be reviewed and then the invoice can
+be payed using an available payment provider app (e.g., banking app).
+
+The Gini Pay API provides an information extraction service for analyzing health invoices. Specifically it extracts
```

Comment:
- "," after Specifically is missing
-  "payed"-> paid 
-  maybe using any available payment provider app (e.g., banking app)?